Brandon_hopper.asm

```diff
@@ -0,0 +1,228 @@
+
+	      .text
+	      
+print:	add  $t0, $zero, $a0  # starting address of array of data to be printed
+	      add  $t1, $zero, $a1  # initialize loop counter to array size
+	      la   $a0, head        # load address of the print heading string
+	      li   $v0, 4           # specify that you're printing a string
+	      syscall               # print the heading string
+
+out:	  lw   $a0, 0($t0)      # load the integer to be printed (the current Fib. number)
+	      li   $v0, 1           # specify that you're printing an integer
+	      syscall               # print fibonacci number
+
+	      la   $a0, space       # load address of spacer for syscall
+	      li   $v0, 4           # specify Print String service
+	      syscall               # print the spacer string
+
+	      addi $t0, $t0, 4      # increment address of data to be printed
+	      addi $t1, $t1, -1     # decrement loop counter
+	      bgtz $t1, out         # repeat while not finished
+
+	      jr   $ra              # return from subroutine
+	# End of subroutine to print the numbers on one line
+	###############################################################
+
+
+##########################################################
+# .data segment for Problem 2
+##########################################################
+.data
+
+Print_Info: .asciiz "\n\n####################\nStarting Problem 2\n##################\nName: Brandon Hopper\nTitle: Printing Program\nDescription: This program was designed to print stuff xD." 
+Ask_input1: .asciiz "\nPlease enter a number for a: "
+Ask_input2: .asciiz "\nPlease enter a number for b: "
+Inser_data: .word 4 #Load word into memory
+
+
+#---------------------------------------------------------
+# Problem # 2
+#---------------------------------------------------------
+.text
+	#Print the info to the screen
+problem2: la $a0, Print_Info
+	li $v0, 4
+	syscall
+	
+	#Print message to ask for user input 1
+	li $v0, 4
+	la $a0, Ask_input1
+	syscall
+		
+	#Prompt for user input 1
+	li $v0,5
+	syscall
+        move $t0, $v0
+        
+	#Print message to ask for user input 2
+	li $v0, 4
+	la $a0, Ask_input2
+	syscall
+		
+	#Prompt for user input 2
+	li $v0,5
+	syscall
+        move $t1, $v0
+        
+        li $v0, 1
+        li $t1, 4
+	syscall
+        
+        add $t3,$zero,$t0 #set t3 to t1s value
+        add $t4,$zero,$t1 #set t4 to t0s value
+        
+        #Multiply and save the results
+mult:   add $t3,$t3,$t3
+	addi $t4, $t4, -1     # decrement loop counter
+	bgtz $t4, mult        # repeat while not finished
+        
+        #Add and set
+	add $t5,$t1,$t0
+	
+	#Subtract and set
+	sub $t6,$t1,$t0
+                 
+        move $a0, $t4
+        #add $a0,$t1,$t0
+        #print the values
+        li $v0, 1
+        li $a0, 4
+	syscall
+	
+	#print the values
+       # li $v0, 1
+       # li $t5, 5
+	#syscall
+	
+	#print the values
+       # li $v0, 1
+       # li $t1, 5
+	#syscall
+	
```

Comment:
-4 Not printing your addition subtraction results, no points lost for multiplication printing, even though its the wrong value.